packages/main/src/themes/SliderBase.css

```diff
@@ -11,6 +11,7 @@
 	cursor: pointer;
 	vertical-align: top;
 	width: 100%;
+	padding: 1rem;
```

Comment:
How do we decide 1rem top and bottom padding? I know that it was done in the following way before your change but as the change name is "adjust padding to spec" I am wondering, because I did not find anything about padding in the specifications and if we look at UI5 as a reference, the padding is 1rem in compact, but in Cozy it is **padding: 1.406rem 1.0625rem**; 
Also for the left and right one, both in compact and cozy currently in UI5 it is 17px(1.0625rem)
The only thing that I found out in the specification is that the touch aria is 3rem high in cozy and 2rem in compact and that is valid for Belize.

Reply:
I have contacted VD to add it to the Fiori 3 spec. No specific number currently exists there, so I've taken into account the radius of the handles.

Reply:
Updated to match paddings from UI5, both in Cozy and Compact.

Reply:
the change is good to be merged but I can't reopen the pull request in result of the master branch name change, so I uploaded a new one